tables/automl/notebooks/retail_product_stockout_prediction/README.md

```diff
@@ -0,0 +1,371 @@
+Click <+New Dataset> → Dataset Name <StockOut> → Click Create Dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%202%202019-03-13%20at%201.05.17%20PM.png" width="50%" height="50%">\
+
+4. You can import data from BigQuery or GCS bucket \
+    a. For BigQuery enter your GCP project ID, Dataset ID and Table ID \
+    After specifying dataset click import dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%203%202019-03-13%20at%201.08.44%20PM.png" width="50%" height="50%">\
+    b. For GCS enter the GCS object location by clicking on BROWSE \
+    After specifying dataset click import dataset \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%204%202019-03-13%20at%201.09.56%20PM.png" width="50%" height="50%">\
+    Depending on the size of the dataset this import can take some time.
+
+5. Once the import is complete you can set the schema of the imported dataset based on your business understanding of the data \
+    a. Select Label i.e. Stockout \
+    b. Select Variable Type for all features \
+    c. Click Continue \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%206%202019-03-13%20at%201.20.57%20PM.png" width="50%" height="50%">\
+
+6. The imported dataset is now analyzed \
+This helps you analyze the size of your dataset, dig into missing values if any, calculate correlation, mean and standard deviation. If this data quality looks good to you then we can move on to the next tab i.e. train. \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%201%202019-03-25%20at%2012.43.13%20AM.png" width="50%" height="50%">\
+
+7. Train \
+    a. Select a model name \
+    b. Select the training budget \
+    c. Select all features you would like to use for training \
+    d. Select optimization objectives. Such as: ROC, Log Loss or PR curve \
+    (As our data is imbalances we use PR curve as our optimization metric) \
+    e. Click TRAIN \
+    f. Training the model can take some time \
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%208%202019-03-13%20at%201.34.08%20PM.png" width="50%" height="50%">\
+<img src="https://storage.cloud.google.com/cloud-ml-data/automl-tables/notebooks/automl_stockout_img/Image%20new%202%202019-03-25%20at%2012.44.18%20AM.png" width="50%" height="50%">\
+
+8. Once the model is trained you can click on the evaluate tab \
+This tab gives you stats for model evaluation  \
+         For example our model shows  \
+         Area Under Precision Recall Curve: 0.645  \
+         Area Under ROC Curve: 0.893 \
+         Accuracy: 92.5% \
+         Log Loss: 0.217 \
+Selecting the threshold lets you set a desired precision and recall on your predictions. \
```

Comment:
remove `< img src="` and replace with `![alt text](` 
and replace  `width="50%"` height="50%">` with `)`